doc/licenses/licenses.xml

```diff
@@ -3500,16 +3500,6 @@
               <url>http://opensource.org/licenses/mit-license</url>
             </license>
                   </licenses>
-      </dependency>
-          <dependency>
-        <packageName>codemirror</packageName>
-        <version>5.62.0</version>
-        <licenses>
-                      <license>
-              <name>MIT</name>
-              <url>http://opensource.org/licenses/mit-license</url>
-            </license>
-                  </licenses>
```

Comment:
Aren't we using `codemirror` anymore? `@types/codemirror` it's in `package.json`. Doesn't it require an item in this list?

Reply:
We continue using `codemirror` but we had them duplicated. You can see at the `yarn.lock` that this dependency is deleted. Nonetheless, we maintain the types since we continue using `codemirror` (you can check that at the `_form.html.slim` partial).